app/connectors/BackendConnector.scala

```diff
@@ -82,7 +82,12 @@ class BackendConnector @Inject() (
   )(implicit hc: HeaderCarrier): Future[Application] =
     httpClient
       .get(url"$backendUrl/applications/$applicationId")
-      .execute[Application]
+      .execute[HttpResponse]
+      .map {
+        response =>
+          val ret = response.json.as[Application]
+          ret
+      }
```

Comment:
If we're not reverting this, it might be worth changing this method to return an Either of Application so that if there is an issue with getting an application we can handle the failure case